pkgs/by-name/bo/boehmgc/package.nix

```diff
@@ -76,10 +76,9 @@ stdenv.mkDerivation (finalAttrs: {
       Alternatively, the garbage collector may be used as a leak detector for
       C or C++ programs, though that is not its primary goal.
     '';
-    # non-copyleft, X11-style license
     changelog = "https://github.com/ivmai/bdwgc/blob/v${finalAttrs.version}/ChangeLog";
-    license = "https://hboehm.info/gc/license.txt";
-    maintainers = with maintainers; [ AndersonTorres ];
-    platforms = platforms.all;
+    license = "https://hboehm.info/gc/license.txt"; # non-copyleft, X11-style license
```

Comment:
How can I create a license for this guy?

Reply:
This seems fine to me, or do you think it should be different?

Reply:
My memory is bad today to catch links.
I remember an issue about those license links. Since they do not have a full structure like the ones in licenses file, they were being treated as "free by default".